workflows/video-generation-workflow/README.md

```diff
@@ -0,0 +1,50 @@
+# Spell Workflow Example: Video Generation
+This is an example of how to use the Spell Python API and a workflow to train a [pix2pix](https://github.com/affinelayer/pix2pix-tensorflow) model and generate videos with it.
+
+For more information regarding Spell, the Python API, runs, and workflows,
+check out the [Spell documentation](https://spell.run/docs).
+
+To run this code you first need to [sign up for a Spell account](https://web.spell.run/register)
+and install Spell: `pip install spell`.
+
+# Introduction
+Pix2pix is an image-to-image transformation method. Given a video, we can train a pix2pix model to predict the next frame based on current frame. After training, such a model could be used to generate a video starting from only one frame. In the generation process, every newly created frame would be fed back to the model to predict the next one.
+
+# Run it
+
+### 1. Clone the [Video Generation](https://github.com/chengz3906/video-generation) repo:
+```ShellSession
+$ git clone https://github.com/chengz3906/video-generation.git
```

Comment:
i feel like this would be sweeter by not requiring these repo clones by using the github url as the repo instead of a path to a clone.

Reply:
Both are fixed now :)

Reply:
And I just walked through it. Didn't encounter problems.